pkg/log/level.go

```diff
@@ -0,0 +1,65 @@
+package log
+
+import (
+	"io"
+	"io/ioutil"
+	"os"
+)
+
+const (
+	OFF = iota
+	DEBUG
+	INFO
+	WARNING
+	ERROR
+)
+
+const (
+	MinLevel = OFF
+	MaxLevel = ERROR
+)
+
+func GetPrefixForLogger(level int) string {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported function GetPrefixForLogger should have comment or be unexported](https://app.codacy.com/app/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4570750)